src/App.scss

```diff
@@ -1,42 +1,56 @@
 @import './utils/variables';
-
+html,
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+html {
+    -ms-overflow-style: none;
+    scrollbar-width: none;
+    overflow-x: hidden;
+    overflow-y: scroll;
```

Comment:
Hmm 🤔  @tunangst looking at staging now we don't have a y-axis scroll bar. Do you think this somehow lead to that?

Reply:
It is. The page should still scroll. We could revert if we want it back for cases where people don't have a scroll or know how to move the page without one, the alternative would be making the whole screen things like the modal have a scroll bar so the styling doesn't shift and always keeps the bar there, or the other alternative if we don't really care about the problem.
There have just been a few times where I have been testing some code and trying to click on something and the styling shifts over half an inch due to the scroll being added or removed by the browser. This was pretty annoying and looks a little tacky. 🍔